### PR TITLE
[PAY-945][PAY-962][PAY-948][PAY-977] More gated content QA fixes

### PR DESCRIPTION
**Pull request by sddioulde**
### Description

- allow pasting link for track remix even without switching remix toggle on
- fix dog ear shadow clipping
- add breakpoint for how to unlock section for smaller screens
- fix learn more link style

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

local v. stage

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

existing premium content feature flags

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-gated-qa

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-gated-qa

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-gated-qa

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-gated-qa

**Comment by sddioulde** [comment]
there is some pretty similar logic in some of these components. will do a refactor pass later

File: packages/mobile/src/components/details-tile/DetailsTile.tsx [hunk 1]
```diff
@@ -222,14 +226,20 @@ export const DetailsTile = ({
...
-          isOwner={isOwner}
-          premiumConditions={premiumConditions}
-        />
-      )
+    const showPremiumCornerTag =
+      isPremiumContentEnabled &&
+      premiumConditions &&
+      (isOwner || !doesUserHaveAccess)
+    const cornerTagIconType = showPremiumCornerTag
+      ? isOwner
```
Comment (dharit-tan) [inline packages/mobile/src/components/details-tile/DetailsTile.tsx:234]: Just making sure here - does this logic translate to:
if isOwner, show lock icon
else show nothing
Or am I misreading this?

Comment (dharit-tan) [reply packages/mobile/src/components/details-tile/DetailsTile.tsx:234]: Ah no I think I'm wrong - if isOwner, enter the next branch, else show lock.

File: packages/web/src/components/track/TrackBannerIcon.tsx [hunk 2]
```diff
@@ -7,36 +8,52 @@ import styles from './TrackBannerIcon.module.css'
...
+  LOCKED = 'locked',
+  COLLECTIBLE_GATED = 'collectible gated',
+  SPECIAL_ACCESS = 'special access'
 }
 
 const TrackBannerIcon = ({
   type,
   isMobile,
-  isMatrixMode
+  isMatrixMode,
```
Comment (dharit-tan) [inline packages/web/src/components/track/TrackBannerIcon.tsx:20]: Feels like we shouldn't have to do this.. is this something that happens elsewhere as well? Ok to merge this for now but should maybe investigate further.

**Review by dharit-tan — APPROVED** [review]

File: packages/mobile/src/components/details-tile/DetailsTile.tsx (hunk 1 above)
Comment (sddioulde) [reply packages/mobile/src/components/details-tile/DetailsTile.tsx:234]: it translates to:
- if owner, then see the premium conditions
  - if collectible gated, then show collectible gated that icon
  - if special access, then show special access icon
- if not owner, then show locked icon

File: packages/web/src/components/track/TrackBannerIcon.tsx (hunk 2 above)
Comment (sddioulde) [reply packages/web/src/components/track/TrackBannerIcon.tsx:20]: hmm not sure why you think we shouldn't have to do this. this component is used by different other components, and we sometimes need to override

**Pull request merged**